package.json

```diff
@@ -11,9 +11,9 @@
   "scripts": {
```

Comment:
Seems like this would also be an excellent time to pull the `.d.ts` from the UI into here, until this is fully converted, and add a `types` here, so we can start adding and changing types here and immediately pull the result through elsewhere.